react/ActionMenu/index.jsx

```diff
@@ -1,54 +1,62 @@
-import React, { useCallback, useContext } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
 import ClickAwayListener from '@material-ui/core/ClickAwayListener'
 import styles from './styles.styl'
 import { Media, Bd, Img } from '../Media'
 import BottomDrawer from '../BottomDrawer'
 import withBreakpoints from '../helpers/withBreakpoints'
-import Popper from '@material-ui/core/Popper'
 import { getCssVariableValue } from '../utils/color'
-import PopperContainerContext from '../PopperContainerContext'
 import Radio from '../Radio'
 import { spacingProp } from '../Stack'
+import { usePopper } from 'react-popper'
```

Comment:
for information: 
BUNDLE SIZE
5.2kB

2kBMINIFIED + GZIPPED

https://bundlephobia.com/result?p=react-popper@2.2.3